testing/AirSimTests/log_analyzer.py

```diff
@@ -1,14 +1,18 @@
+import json
 import sys
+from types import SimpleNamespace
 from datetime import datetime
 from pathlib import Path
-from typing import List, Tuple, Dict, Optional
+from typing import List, Tuple, Optional
 import MapSerializer as MapSerializer
-from Models import Point as PointModel, RoadSegmentType
 from PIL import Image, ImageDraw, ImageColor
-from sympy import Point2D, Point, Segment
+from sympy import Point, Segment
+from common.models import LogEntry
+from Models import X_COORD, Y_COORD, point_to_gui_coords
 
 TARGET_AREA = 100.0
-MAX_STEERING = 1.0
+TARGET_TIME = 150
+MAX_STEERING = 0.20
```

Comment:
Why do we change max steering to 0.2 and not leave it as 1.0?

Reply:
The steering never exceeds 1.0, so that was just a placeholder value. 0.20 is the target metric from the proposal, but it seems too sensitive as it always triggers in corners.

Reply:
I see, i guess we'll have to figure out the value afterward right?

Reply:
I mean isn't steering suppose to exceed 0.2 at intersections as the car is making a turn, I assumed we'd only be looking at steering spikes past our intended goal for only striaght roads?

Reply:
No longer marks turns as exceeding max steering